api/views/pois.py

```diff
@@ -6,6 +6,9 @@
 from datetime import date
 from dateutil.parser import parse
 
+POIS_URL = '/pois'
+POIS_ID_URL = '/pois/<poi_id>'

```

Comment:
added one more thing to the [conventions](https://github.com/hack4impact-uiuc/wiki/wiki/Flask-API-Conventions#7-cast-ids-to-int-in-the-endpoint-url) - cast to int in the url itself. and make sure to update where its used